### DISCUSSION TITLE
[Suggestion] Add Positions and Weekly Leaderboard

### DISCUSSION THREAD

**Discussion by ColonialIII** [HypixelDev/PublicAPI#304] [Ideas]
I have two suggestions here to offer, both would be extremely useful for me personally and I'm sure a lot of other people using the API.

Suggestion #1: Add a Daily Leaderboard to the Leaderboard API. This is mainly for Bedwars, as there is already a Weekly Leaderboard, but not a Daily...

Suggestion #2: Add positions to the Player API. For example I'm currently 213th for Final Kills with 30,000. Make a way to show this positionin the Player's Stats API.

**Comment by GoogleSites** [comment]
+1 for the first suggestion.
-1 for the second suggestion. Calculating the player's position for each statistic would take a HUGE hit on their database as they'd need to sort and find the player (out of >16 million players present) many, many times over. I could see this in another endpoint (eg. `/position`) with some parameters to fetch their position though.
